sqlglot/expressions.py

```diff
@@ -2623,6 +2631,10 @@ class GroupConcat(Func):
     arg_types = {"this": True, "separator": False}
 
 
+class Hex(Func):
+    arg_types = {"this": True}
```

Comment:
We can `pass` here, because `arg_types` is already defined like this in `Expression` (same for `Unhex`).

Reply:
good point,thanks